share/picongpu/tests/KHI_growthRate/README.rst

```diff
@@ -0,0 +1,39 @@
+KelvinHelmholtz: Kelvin-Helmholtz Instability
+=============================================
+
+.. sectionauthor:: Richard Pausch <r.pausch (at) hzdr.de>
+.. moduleauthor:: Richard Pausch <r.pausch (at) hzdr.de>, Mika Soren Voß <m.voss (at) hzdr.de>
+
+This KHI growth rate test simulates a shear-flow instability known as the Kelvin-Helmholtz Instability in a sub-relativistic setup as studied in [Alves12]_, [Grismayer13]_, [Bussmann13]_. The setup uses a pre-ionized quasi-neutral hydrogen plasma. From the simulation, the growth of the dominant magnetic field is extracted and compared to theoretic predictions. The tests only passes if the simulation reaches the ideal theoretical growth rate with a 10% accuracy. 
+
+
+Usage
+-----
+
+.. code-block:: bash
+
+  # execute ci
+  $PICSRC/share/picongpu/tests/KHI_growthRate/bin/ci.sh $PICSRC/share/picongpu/tests/KHI_growthRate/ <user_given_output_directory>
+
+
+References
+----------
+
+.. [Alves12]
+       E.P. Alves, T. Grismayer, S.F. Martins, F. Fiuza, R.A. Fonseca, L.O. Silva.
+       *Large-scale magnetic field generation via the kinetic kelvin-helmholtz instability in unmagnetized scenarios*,
+       The Astrophysical Journal Letters (2012),
+       https://dx.doi.org/10.1088/2041-8205/746/2/L14
+
+.. [Grismayer13]
+       T. Grismayer, E.P. Alves, R.A. Fonseca, L.O. Silva.
+       *dc-magnetic-field generation in unmagnetized shear flows*,
```

Comment:
Phys Rev writes the title as 'dc-Magnetic-Field Generation in Unmagnetized Shear Flows'.